package.json

```diff
@@ -1,46 +1,19 @@
 {
```

Comment:
Can you explain why we still have a top level package.json and a top level yarn.lock?

Reply:
We shouldn't need them anymore. That's leftover from when Thorsten and I were trying to keep some common dependencies without the workspace. I'll remove and we'll see if CI still passes.

Reply:
Actually, we still need the root `package.json` because the base amman config relies on it. We can create an amman config for each package and remove the root amman config and `package.json` files later in a separate PR if we decide to go that route.